site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -352,11 +352,16 @@
                     "content" : upload_message
                 } only %}
             </div>
-            <button type="button" id="submit" class="submit-gradeable btn btn-success">
-                {{ viewing_inactive_version ? 'Switch To Most Recent Version To Make New Submission' : 'Submit' }}
+            <button type="button" id="submit" class="submit-gradeable btn btn-success" {% if viewing_inactive_version %} title="Switch to most recent version to submit"{% endif %}>
+                Submit
+            </button>
+        {% if viewing_inactive_version %}
+            <button type="button" id="switch_to_most_recent" class="btn btn-success" onclick="switchToMostRecent('{{ recent_version_url }}')">
+                Switch to Most Recent Version
             </button>
```

Comment:
I think we want to use a `a` tag here instead of a button for better accessibility as it's an anchor to another page, versus doing some action on the current page.

```suggestion
            <a href='{{ recent_version_url }}' class='btn btn-success' id='switch-to-most-recent'>
                Switch to Most Recent Version
            </a>
```

This would then allow getting rid of the added JS as well.